update: iOS 18 Browser Recommendations Improvements (#2802)
privacyguides/privacyguides.org: docs/mobile-browsers.md

## Safari (iOS)

> On iOS, any app that can browse the web is [restricted](https://developer.apple.com/app-store/review/guidelines) to using an Apple-provided [WebKit framework](https://developer.apple.com/documentation/webkit), so a browser like [Brave](#brave) does not use the Chromium engine like its counterparts on other OSes.

Reviewer: ```suggestion
On iOS, any app that can browse the web is [restricted](https://developer.apple.com/app-store/review/guidelines) to using an Apple-provided [WebKit framework](https://developer.apple.com/documentation/webkit), so a browser like [Brave](#brave) does not use the Chromium engine like its counterparts on other operating systems.
```
Suggesting to expand the abbreviation since it's the first instance of the term on the page

The following privacy/security-related options can be found in :gear: **Settings** → **Apps** → **Safari**.

#### Allow Safari to Access

Under **Siri**:

- [ ] Disable **Learn from this App**
- [ ] Disable **Show in App**
- [ ] Disable **Show on Home Screen**
- [ ] Disable **Suggest App**

This prevents Siri from using content from Safari for Siri suggestions.

#### Search

- [ ] Disable **Search Engine Suggestions**

> This setting sends your search queries before you press enter, which might cause you to send data you didn't intend to.

Reviewer: ```suggestion
This setting sends whatever you type in the address bar to the search engine set in Safari. Disabling search suggestions allows you to more precisely control what data you send to your search engine provider.
```

This setting allows you to lock your private tabs behind biometrics/PIN when not in use.

- [ ] Disable **Fraudulent Website Warning**

> This setting uses Google Safe Browsing (or Tencent Safe Browsing for users in mainland China or Hong Kong) to protect you while you browse. Disabling this setting is more private but you will sacrifice some security.

Reviewer: ```suggestion
This setting uses Google Safe Browsing (or Tencent Safe Browsing for users in mainland China or Hong Kong) to protect you while you browse. As such, your IP address may be logged by your Safe Browsing provider. Disabling this setting will disable this logging, but you will sacrifice some security.
```

- [ ] Disable **Highlights**

> >When visiting a webpage, Safari may send information calculated from the webpage address to Apple over OHTTP to determine if relevant highlights are available.

Reviewer: ```suggestion
Apple's privacy policy for Safari states:

> When visiting a webpage, Safari may send information calculated from the webpage address to Apple over OHTTP to determine if relevant highlights are available.
```

> -[Apple](https://www.apple.com/legal/privacy/data/en/safari/)

Reviewer: ```suggestion
```
With my suggestion above to introduce the quote, I don't think it's necessary to link the privacy policy (it's already in the card).

#### Settings for Websites

Under **Camera**

- [x] Select **Ask**

Under **Microphone**

- [x] Select **Ask**

Under **Location**

- [x] Select **Ask**

> These settings are important so that websites can't access your camera, microphone, and location whenever they want.

Reviewer: ```suggestion
These settings ensure that websites can only access your camera, microphone, or location after you explicitly grant them access.
```

This setting allows you to lock your private tabs behind biometrics/PIN when not in use.

- [ ] Disable **Fraudulent Website Warning**

> This setting uses Google Safe Browsing (or Tencent Safe Browsing for users in mainland China or Hong Kong) to protect you while you browse. As such, your IP address may be logged by your Safe Browsing provider. Disabling this setting will disable this logging, but you will sacrifice some security.

Reviewer: ```suggestion
This setting uses Google Safe Browsing (or Tencent Safe Browsing for users in mainland China or Hong Kong) to protect you while you browse. As such, your IP address may be logged by your Safe Browsing provider. Disabling this setting will disable this logging, but you might be more vulnerable to known phishing sites.
```
I think it's better to be more specific about the security trade-off of turning off FWW.
